publication/3-cr/index.html

```diff
@@ -1624,13 +1632,22 @@ <h4>Listing</h4>
                                         and `sort_order` to choose the order
                                         (i.e. `asc` or `desc` for ascending and descending ordering).
                                     </span>
+                                    <span class="rfc2119-assertion" id="tdd-things-list-pagination-orderable">
+                                        A server MUST reject requests to sort on fields that do not have 
+                                        values that are orderable basic types, 
+                                        with a 501 (Not Implemented) status.
+                                    </span>
```

Comment:
I think this should be 400 (Bad Request). Not implemented is suitable when the server hasn't implemented a feature by choice.

Reply:
also... that was editing 3-cr/index.html for some reason!  Argh...

Reply:
fixed, moved (as are all other changes in this file...) to index.html